tools/update-webpage.sh

```diff
@@ -44,6 +45,7 @@ titles[$REFERENCE_COUNTING_MD]="Reference Counting"
 titles[$DEBUGGER_MD]="Debugger"
 titles[$CODING_STANDARDS_MD]="Coding Standards"
 titles[$EXT_REFERENCE_ARG_MD]="'Extension API: Argument Validation'"
+titles[$EXT_REFERENCE_AUTORELEASE_MD]="'Extension API: Autorelease Values'"
```

Comment:
I see no reason for listing the titles in a different order than the file names. This line should come last in this list.